app/services/cancel_outstanding_references.rb

```diff
@@ -0,0 +1,17 @@
+class CancelOutstandingReferences
+  attr_reader :application_form
+
+  delegate :application_references, :show_new_reference_flow?, to: :application_form
+
+  def initialize(application_form:)
+    @application_form = application_form
+  end
+
+  def call!
```

Comment:
Do we have a convention where `!` indicates something's getting deleted?

Reply:
We do have some services like cancel interviews that uses